dapper/cmd/dapper-db-meta-ingest/main.go

```diff
@@ -172,43 +172,45 @@ func main() {
 				}
 			}
 
-			for _, l := range km.Relations {
+			for toKey, rs := range km.Relations {
 				// Makes sure if the keys exists
 				found := false
 				for _, k := range input.Metadata {
-					if l.Relation.FromKey == k.Key {
+					if km.Key == k.Key {
 						found = true
 					}
 				}
```

Comment:
km is from input.Metadata already, so comparing to k (also from input.Metadata) is weird, won't it always be found?

Reply:
Nice catch. Should remove it. Thanks.